File: js/ejer43.js
//encontrar el nunmero mayor de un arreglo
let arreglo = [1, 20, 3, 4, 50];
let mayor = arreglo[0];

for (let i = 1; i < arreglo.length; i++) {
    if (arreglo[i] > mayor) {
        mayor = arreglo[i];
    }
}

console.log("El número mayor es: " + mayor);

File: python/ejer43.py
#encontrar el nunmero mayor de un arreglo
def mayor(arreglo):
    mayor=arreglo[0]
    for i in range(1,len(arreglo)):
        if arreglo[i]>mayor:
            mayor=arreglo[i]
    return mayor
arreglo=[1,20,3,4,5]
print(mayor(arreglo))